### PR TITLE
Add puppet-lint plugins from deanwilson to plugins page

### PR DESCRIPTION
**Pull request by deanwilson**
I've added a small collection of my public puppet-lint plugins to the plugins page. Hopefully some of them will be of use to other people.

### CODE REVIEW

File: plugins/index.md [hunk 1]
```diff
@@ -112,3 +112,104 @@ layout: default
...
+
+---
+
+
+### duplicate_class_parameters
+
+> Ensures class parameter names are unique.
+
+| **URL**     | <https://github.com/deanwilson/puppet-lint_duplicate_class_parameters-check> |
+| **Install** | `gem install puppet-lint_duplicate_class_parameters-check`                   |
```
Comment (rnelson0) [inline plugins/index.md]: (or the gemspec/rubygem needs renamed)

File: plugins/index.md [hunk 2]
```diff
@@ -112,3 +112,104 @@ layout: default
...
+> Ensure file resources do not have a path attribute.
+
+| **URL**     | <https://github.com/deanwilson/puppet-lint-no_file_path_attribute-check> |
+| **Install** | `gem install puppet-lint-no_file_path_attribute-check`                   |
+{: .table .table-condensed }
+
+---
+
+
+### non_erb_template_filename
```
Comment (rnelson0) [inline plugins/index.md]: Not related to this PR, but would it be helpful to remove `erb` from the name and have the check look at epp filenames as well?

**Comment by rnelson0** [comment]
@deanwilson This looks good once the typo is addressed. That's quite a collection of plugins you have! Thanks for creating them.

**Comment by rnelson0** [comment]
@deanwilson Have you had a chance to review and update this? Thanks!

**Comment by binford2k** [comment]
I corrected the typo, and am merging as-is. It would be great to update the template filename check as @rnelson0 suggests though!

**Pull request merged**